mcp_dispatcher.py

```
@@ -125,14 +139,51 @@ def main() -> None:
+                    flush=True,
+                )
+                last_seen = created_at or datetime.now(timezone.utc)
+                save_last_seen(last_seen.isoformat())
+                continue
+            owned_agent_id = item_meta.get("agent_id")
+            if item and action and owned_agent_id:
+                owner = find_member(owned_agent_id)
+                if owner:
+                    try:
+                        result = owner.execute_action(item, action)
+                    except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)